hack/lib/serverless.bash

```diff
@@ -92,23 +92,21 @@ function find_install_plan {
   echo ""
 }
 
-function deploy_knativeserving_cr {
+# Deploys the version of KnativeServing from 1.3.0 release. The future releases should ensure compatibility with
+# this custom resource and accept it.
+function deploy_knativeserving_v1alpha1_1.3.0 {
   logger.info 'Deploy Knative Serving'
-  local group=${1:-operator}
 
   # Wait for the CRD to appear
   timeout 900 "[[ \$(oc get crd | grep -c knativeservings) -eq 0 ]]" || return 6
 
+  local rootdir
+  rootdir="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
+
   # Install Knative Serving
-  cat <<EOF | oc apply -f - || return $?
-apiVersion: ${group}.knative.dev/v1alpha1
-kind: KnativeServing
-metadata:
-  name: knative-serving
-  namespace: ${SERVING_NAMESPACE}
-EOF
+  oc apply -n "${SERVING_NAMESPACE}" -f "${rootdir}/test/v1alpha1/resources/serving.knative.dev_v1alpha1_knativeserving_cr_1.3.0.yaml" || return $?
 
-  timeout 900 '[[ $(oc get knativeserving.${group}.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.conditions[?(@.type==\"Ready\")].status}") != True ]]'  || return 7
+  timeout 900 '[[ $(oc get knativeserving.serving.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.conditions[?(@.type==\"Ready\")].status}") != True ]]'  || return 7
```

Comment:
Right now, the behavior of the operator is to immediately convert and then delete an old CR when it detects one, so the old CR's status will never become ready.